### DISCUSSION TITLE
Next.js docs tutorial (dashboard project) not deploying to Vercel

### DISCUSSION THREAD

**Discussion by JvR96** [vercel/community#4930] [Help] [answered]
### Summary

Hi all, I am currently learning Next.js using their docs and the associated Dashboard follow-along project. I am now at Chapter 6: Setting up your database (https://nextjs.org/learn/dashboard-app/setting-up-your-database).

I have successfully connected my GitHub project to Vercel and my inital commit is also correctly showing under the generated domain (https://nextjs-docs-dashboard-abvb.vercel.app/).

However, any new commits to my project are no longer being deployed to Vercel. When pushing changes to GitHub, I can see that Vercel is trying to deploy the changes, but I get the following error message: "An unexpected error happened when running this build. We have been notified of the problem. This may be a transient error. If the problem persists, please contact Vercel Support https://vercel.com/help"

Does anyone have experience with this? Any help is greatly appreciated! 

Thank you,
Jannick

### Example

https://nextjs-docs-dashboard-abvb.vercel.app/

### Steps to Reproduce

Error message on deploy to Vercel: "An unexpected error happened when running this build. We have been notified of the problem. This may be a transient error. If the problem persists, please contact Vercel Support https://vercel.com/help"

**Comment by swarnava** [comment, marked as answer]
Hi,

This sounds like a transient error. Are you still facing issue?

**Reply by JvR96** [reply]
Hi swarnava, yes I now see that this error has been resolved. Do you know why it came to be in the first place? Thank you for your reply!

**Reply by swarnava** [reply]
Hi, we had a relevant incident yesterday for 30 mins which seems life affected the build time. This issue should no longer happen.
